src/category/bitraversable/instances.lean

```diff
@@ -1,8 +1,26 @@
+/-
+Copyright (c) 2019 Simon Hudon. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author(s): Simon Hudon
+-/
+
 import category.bitraversable.basic
        category.bitraversable.lemmas
        category.traversable.lemmas
        tactic.solve_by_elim
 
+/-!
```

Comment:
Could you structure this according to the guidelines, with a proper title, sections, tags, etc?